generator/standalone_main.cc

```diff
@@ -314,8 +313,70 @@ int main(int argc, char** argv) {
       return google::cloud::Status{};
     }));
   }
+  return tasks;
+}
+
+}  // namespace
+
+/**
+ * C++ microgenerator.
+ *
+ * @par Command line arguments:
+ *  --config-file=<path> REQUIRED should be a textproto file for
+ *      GeneratorConfiguration message.
+ *  --protobuf_proto_path=<path> REQUIRED path to .proto files distributed with
+ *      protobuf.
+ *  --googleapis_proto_path=<path> REQUIRED path to .proto files distributed
+ *      with googleapis repo.
+ *  --output_path=<path> OPTIONAL defaults to current directory.
+ */
+int main(int argc, char** argv) {
+  absl::ParseCommandLine(argc, argv);
+  google::cloud::LogSink::EnableStdClog(
+      google::cloud::Severity::GCP_LS_WARNING);
+
+  CommandLineArgs const args{absl::GetFlag(FLAGS_config_file),
+                             absl::GetFlag(FLAGS_protobuf_proto_path),
+                             absl::GetFlag(FLAGS_googleapis_proto_path),
+                             absl::GetFlag(FLAGS_golden_proto_path),
+                             absl::GetFlag(FLAGS_output_path),
+                             absl::GetFlag(FLAGS_scaffold_templates_path),
+                             absl::GetFlag(FLAGS_scaffold),
+                             absl::GetFlag(FLAGS_experimental_scaffold),
+                             absl::GetFlag(FLAGS_update_ci)};
+
+  GCP_LOG(INFO) << "proto_path = " << args.protobuf_proto_path << "\n";
+  GCP_LOG(INFO) << "googleapis_path = " << args.googleapis_proto_path << "\n";
+  GCP_LOG(INFO) << "config_file = " << args.config_file << "\n";
+  GCP_LOG(INFO) << "output_path = " << args.output_path << "\n";
+
+  auto config = GetConfig(args.config_file);
+  if (!config.ok()) {
+    GCP_LOG(ERROR) << "Failed to parse config file: " << args.config_file
+                   << "\n";
+  }
```

Comment:
It looks like there should be something like `return 1` here, otherwise we go on to use `*config`.